apm-agent-plugins/apm-grpc/apm-grpc-plugin/pom.xml

```diff
@@ -37,7 +37,12 @@
             <version>${grpc.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty-shaded</artifactId>
+            <version>${grpc.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
[question] why do we need this extra dependency for testing ? is it required only for newer versions ?

Reply:
I used this for getting sources so I can debug an entire version. Since it's in `test` scope, I think worth keeping it for the same purpose.